enterprise/migrations/0027_auto_20170921_1201.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0026_make_require_account_level_consent_nullable'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='enrollmentnotificationemailtemplate',
+            name='enterprise_customer',
+            field=models.OneToOneField(related_name='enterprise_enrollment_template', default=None, to='enterprise.EnterpriseCustomer'),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='historicalenrollmentnotificationemailtemplate',
+            name='enterprise_customer',
+            field=models.ForeignKey(related_name='+', on_delete=django.db.models.deletion.DO_NOTHING, db_constraint=False, blank=True, to='enterprise.EnterpriseCustomer', null=True),
+        ),
+        migrations.AlterField(
```

Comment:
@douglashall Do we have to do the whole safe-deployment shenanigans for altering fields too?